tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/SimpleTclGeneratorIntegrationTest.xtend

```diff
@@ -91,28 +92,29 @@ class SimpleTclGeneratorIntegrationTest extends AbstractTclGeneratorIntegrationT
 			
 			import org.junit.Test;
 			import org.testeditor.dsl.common.testing.DummyFixture;
+			import org.testeditor.fixture.core.AbstractTestCase;
 			
 			/**
 			 * Generated from SimpleTest.tcl
 			 */
 			@SuppressWarnings("all")
-			public class SimpleTest {
+			public class SimpleTest extends AbstractTestCase {
 			  private DummyFixture dummyFixture = new DummyFixture();
 			  
 			  @Test
 			  public void execute() throws Exception {
 			    
 			    /* Test assertions in the famous greeting application */
 			    
-			    // Component: GreetingApplication
+			    logger.trace(" [Component] GreetingApplication");
 			    
-			    // - Read list from <bar>
+			    logger.trace(" [test step] - Read list from <bar>");
 			    java.util.List<? extends java.lang.Object> foo = dummyFixture.getList("label.greet");
-			    // - Read value from <bar>
+			    logger.trace(" [test step] - Read value from <bar>");
 			    java.lang.String baz = dummyFixture.getValue("label.greet");
-			    // - Read bool from <bar>
+			    logger.trace(" [test step] - Read bool from <bar>");
 			    boolean book = dummyFixture.getBool("label.greet");
-			    // - Read map from <bar>
+			    logger.trace(" [test step] - Read map from <bar>");
```

Comment:
The assignment should be part of the log message. I know it wasn't part of the comment (yet). It should be though.